botorch/utils/transforms.py

```diff
@@ -154,7 +187,7 @@ def t_batch_mode_transform(
 
     def decorator(method: Callable[[Any, Tensor], Any]) -> Callable[[Any, Tensor], Any]:
         @wraps(method)
-        def decorated(cls: Any, X: Tensor, **kwargs: Any) -> Any:
+        def decorated(cls: Any, X: Tensor, *args: Any, **kwargs: Any) -> Any:
```

Comment:
@Balandat, Should this also change to `acqf: AcquisitionFunction` since that is the only use case? I think it makes sense to keep the signature consistent with `_verify_output_shape`.